app/views/pages/home.html.erb

```diff
@@ -1,8 +1,135 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="#">
+      InstrumentsBnB
+</a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
 
-<p><%= link_to "Click here to view all instruments", instruments_path  %></p>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto">
+          <li class="nav-item active">
+            <a class="nav-link" href="#">Instruments</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Bookings</a>
+          </li>
+          <li class="nav-item dropdown">
+            <img class="avatar dropdown-toggle" id="navbarDropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false" src="https://previews.123rf.com/images/metelsky/metelsky1809/metelsky180900233/109815470-man-avatar-profile-male-face-icon-vector-illustration-.jpg" />
+            <div class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
+              <a class="dropdown-item" href="#">Action</a>
+              <a class="dropdown-item" href="#">Another action</a>
+              <a class="dropdown-item" data-turbo-method="delete" href="#">Log out</a>
+            </div>
+          </li>
+          <!--li class="nav-item"-->
+          <!--/li-->
+      </ul>
+    </div>
+  </div>
+</div>
 
-<p><%= link_to "Click here to sign in", new_user_session_path %></p>
 
-<p><%= link_to "Click here to register", new_user_registration_path %></p>
+<div class="banner" style="background-color: #363c44">
+  <div class="container first">
+    <h1>Rent musical instruments <br>anywhere!</h1>
+    <div class="instruments-link"><%= link_to "Sign in", new_user_session_path %></div>
+  </div>
+</div>
+
+<div class="banner" style="background-color: whitesmoke">
```

Comment:
instead of using inline-style, add a class for the two colors these use: e.g.
```
class="banner-colorA"
```
and
```
class="banner-colorB"
```
or "colorPrimary" and "colorSecondary"
or use `nth-child(:odd)`/`nth-child(:even)` for the two different banners.

Reply:
Yes, you are rigth! Thanks I will update it